doc/sphinx/Pacemaker_Development/helpers.rst

```diff
@@ -224,17 +224,13 @@ is no ``lib/common/tests/acls`` directory.
 
      check_PROGRAMS = pcmk_acl_required_test
 
-* Double check that ``-I$(top_srcdir)/lib/common`` is present in ``AM_CPPFLAGS``.
-
-* Double check the settings of variables at the top of ``Makefile.am``.  The
-  following block should be present:
+* Double check that ``$(top_srcdir)/mk/tap.mk`` and ``$(top_srcdir)/mk/unittest.mk``
+  are included in the ``Makefile.am``.  These files contain all the flags necessary
+  for most unit tests.  If necessary, individual settings can be overridden like so:
 
   .. code-block:: none
 
-     LDADD = $(top_builddir)/lib/common/libcrmcommon_test.la \
-             -lcmocka
-     AM_CFLAGS = -DPCMK__UNIT_TESTING
-     AM_LDFLAGS = $(LDFLAGS_WRAP)
+     LDADD += $(top_builddir)/lib/pengine/libpe_status_test.la
```

Comment:
There might be clearer examples of overriding individual settings than appending a `.la` file

Reply:
I've updated this with the only other thing we're overriding at the moment, which is AM_CPPFLAGS in one spot.